### ISSUE TITLE
Pystemon update/merge/revision

### ISSUE THREAD

**Issue by elbae** [CIRCL/AIL-framework#365]
Hi, I'm using AIL-framework and I think pystemon should be updated. 
Based on the documentation (HOWTO.md) pystemon should be installed from 
https://github.com/CIRCL/pystemon 
but there is also a pystemon folder in the AIL-framework home 
(https://github.com/AIL-framework/pystemon).
The pystemon.yaml in the suggested version (https://github.com/CIRCL/pystemon) does not crawl in the right way https://gist.github.com/ and I think the configuration from https://github.com/CIRCL/pystemon/pystemon.yaml should be included.
Example:
- [version 1 - https://github.com/CIRCL/pystemon/blob/master/pystemon.yaml](https://github.com/CIRCL/pystemon/blob/master/pystemon.yaml)
```
  gist.github.com:
    enable: no
    archive-url: 'https://gist.github.com/gists'
    archive-regex: '<span><a href="/([0-9]+)">gist'
    download-url: 'https://raw.github.com/gist/{id}'
```
- [version 2 - https://github.com/CIRCL/AIL-framework/blob/master/pystemon/pystemon.yaml](https://github.com/CIRCL/AIL-framework/blob/master/pystemon/pystemon.yaml)
```
  gist.github.com:
    enable: no # not present, should be included
    archive-url: 'https://gist.github.com/discover'
    archive-regex: '<a class="link-overlay" href="https://gist.github.com/([A-Za-z0-9]+/[A-Za-z0-9]+)">'
    download-url: 'https://gist.githubusercontent.com/{gist_user}/{gist_id}/raw/'
```
There are additional working sites in "version 2" which are not included in the suggested version ("version 1").

Moreover, the install.sh script in https://github.com/CIRCL/AIL-framework/blob/master/pystemon/install.sh points to https://github.com/cvandeplas/pystemon.git and I could not find and reference to this installation script in the documentation.

I think that install.sh should be referenced in the documentation (HOWTO.md), the proposed pystemon.yaml ("version 1") updated with the values from "version 2" and https://github.com/CIRCL/AIL-framework/blob/master/pystemon/install.sh updated.

Summing up, it migh be useful to merge the different versions.
What do you think?

**Comment by Terrtia** [comment]
Hi @elbae ,

Thanks for your feedback !

The [pystemon](https://github.com/CIRCL/AIL-framework/tree/master/other_installers/docker/pystemon) folder is used by the docker installer.

You right, this https://github.com/CIRCL/pystemon repository should be used.

The old pystemon folder is still located inside the docker directory. This file should be removed in the future
